app/Entities/Post.php

```diff
@@ -27,13 +27,21 @@ class Post extends Entity
      */
     public function link(?Category $category = null, ?Thread $thread = null)
     {
-        $categorySlug = $category === null ?
-            model('CategoryModel')->find($this->category_id)->slug :
-            $category->slug;
-
-        $threadSlug = $thread === null ?
-            model('ThreadModel')->find($this->thread_id)->slug :
-            $thread->slug;
+        if (! isset($this->category_slug)) {
+            $categorySlug = $category === null ?
+                model('CategoryModel')->find($this->category_id)->slug :
```

Comment:
Let's define a standard from now on model(CategoryModel::class) or model('CategoryModel')?
We are using arbitrary in different sections, this destroys code integrity.

Reply:
Good point. Personally, I like `model(CategoryModel::class)` more and that is a convention mostly used in the codebase I think.